### PR TITLE
Image promotion for cluster-api-helm v0.1.0-alpha.4

### PR DESCRIPTION
**Pull request by Jont828**
Image promotion for cluster-api-helm v0.1.0-alpha.4
This is an automated PR generated from `kpromo`
```
kpromo pr --fork Jont828 --project cluster-api-helm --reviewers "@Jont828 @CecileRobertMichon @jackfrancis" --tag v0.1.0-alpha.4 --image cluster-api-helm-controller
```

/hold
cc: @Jont828 @CecileRobertMichon @jackfrancis

### CODE REVIEW

File: registry.k8s.io/images/k8s-staging-cluster-api-helm/images.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 - name: cluster-api-helm-controller
   dmap:
     "sha256:7408f3e79d04ec94c0677b54b1eea73e65aa5748bf65ddb34ca240bf50dc5e0d": ["v0.1.0-alpha.1"]
+    "sha256:152b9703bc6940edd69ea74b014f9c3dc28042ab2cff79e816e6110885191936": ["v0.1.0-alpha.4"]
```
Comment (CecileRobertMichon) [inline registry.k8s.io/images/k8s-staging-cluster-api-helm/images.yaml:4]: what happened to .2 and .3?

Comment (Jont828) [reply registry.k8s.io/images/k8s-staging-cluster-api-helm/images.yaml:4]: I made a mistake with .2 in the generation of the release artifacts and had a similar issue with .3. So currently I'm iteration on the v0.1.0-alpha.[n].

**Pull request merged**